### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by simonaubertbd** [yudiell/python-postman#22]
Hello,

I'm very, very interested in your python-postman. However, could you add some code examples of how to use it, please?

Best regards,

Simon

**Comment by yudiell** [comment]
Yes, I am working on updating the library and thereby updating the documentation.

**Comment by simonaubertbd** [comment]
@yudiell  Thanks !

**Comment by yudiell** [comment]
I've relased a new version of the library that contains more documentation.

**Comment by simonaubertbd** [comment]
@yudiell thanks you, I was able to make it work ! :)

Best regards,

Simon
